recipe/meta.yaml

```diff
@@ -39,10 +38,9 @@ requirements:
     - pip
     - python
     - requests
-    - scikit-build-core
+    - scikit-build-core >=0.9.0
     - setuptools
   run:
-    - asn1crypto
     - {{ pin_compatible("cffi") }}
```

Comment:
I think you will want to start using that [new environment variable](https://ofek.dev/coincurve/install/#source) I introduced called `COINCURVE_VENDOR_CFFI`. The default behavior is now to ship a shared library from `cffi` during the build step in the wheel which I think might be undesirable here.